lib/react_client.dart

```diff
@@ -275,17 +285,19 @@ ReactComponentFactory _registerComponent(ComponentFactory componentFactory, [Ite
  * create dart-react registered component for html tag.
  */
 _reactDom(String name) {
-  return (Map args, [children]) {
-    _convertBoundValues(args);
-    _convertEventHandlers(args);
-    if (args.containsKey('style')) {
-      args['style'] = new JsObject.jsify(args['style']);
+  var _call = (Map props, [dynamic children]) {
+    _convertBoundValues(props);
+    _convertEventHandlers(props);
+    if (props.containsKey('style')) {
+      props['style'] = new JsObject.jsify(props['style']);
     }
     if (children is Iterable) {
       children = new JsArray.from(children);
     }
-    return _React['createElement'].apply([name, newJsMap(args), children]);
+    return _React['createElement'].apply([name, newJsMap(props), children]);
   };
+
+  return new ReactComponentFactoryProxy(context['React']['DOM'][name], _call);
```

Comment:
Also minor, but this can be `_React['DOM'][name]` instead of `context['React']['DOM'][name]`.